src/components/TextStack/TextStack.test.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import { TextStack, TextStackProps } from './';
+
+import styles from './TextStack.module.scss';
+
+const DEFAULT_PROPS: TextStackProps = {
+  label: '',
+  primaryText: '',
+  secondaryText: undefined
+};
+
+const MOCK_SKELETON_ID = 'mock-skeleton';
+
+jest.mock('../Skeleton', () => ({
+  Skeleton: () => <div data-testid={MOCK_SKELETON_ID} />
+}));
+
+describe('<TextStack />', () => {
+  test('should render in the correct order - label, primary, secondary', () => {
+    const { container } = render(<TextStack label={'label'} primaryText={'primary'} secondaryText={'secondary'} />);
+    expect(container.textContent).toBe('labelprimarysecondary');
+  });
+
+  describe('semantic tags', () => {
+    test('should render label as <label>', () => {
+      const { container } = render(<TextStack {...DEFAULT_PROPS} label={'mock label'} />);
+      const label = container.getElementsByTagName('label');
+      expect(label.length).toBe(1);
+      expect(label[0]).toHaveTextContent('mock label');
+    });
+  });
+
+  describe('class names', () => {
+    test('should apply Container class to container', () => {
+      const { container } = render(<TextStack {...DEFAULT_PROPS} />);
+      expect(container.firstChild).toHaveClass(styles.Container);
```

Comment:
<img width="635" alt="Screenshot 2022-09-30 at 14 06 20" src="https://user-images.githubusercontent.com/39112648/193276168-0bb51b87-fe97-460a-bda1-5620eb25ce3a.png">

I think this applies here?